lib/hexpm/repository/package_reports.ex

```diff
@@ -62,4 +155,38 @@ defmodule Hexpm.Repository.PackageReports do
     PackageReportComment.all_for_report(report_id)
     |> Repo.all()
   end
+
+  defp email_user_about_new_report({:ok, package_report}, user) do
```

Comment:
If you use `Repo.insert!` instead of `Repo.insert` you don't have to do the `{:ok, ...}` match.

Reply:
You should always `Repo.insert!` unless you are actually matching on the result later. Otherwise the insert can fail and we don't know about it.